pkg/resource/deploy/step.go

```diff
@@ -289,6 +289,7 @@ func (s *CreateStep) Apply() (resource.Status, StepCompleteFunc, error) {
 
 	id := s.new.ID
 	outs := s.new.Outputs
+	refreshBeforeUpdate := false
```

Comment:
GitHub doesn't let me comment on unchanged lines, but I'm wondering whether we need to explicitly propagate `old.RefreshBeforeUpdate` to `new.RefreshBeforeUpdate` in the case of a `SameStep`? I think not, but just raising it in case -- @Frassle any thoughts?

Reply:
It will be copied over as part of initializing `new`.